packages/authenticator/amplify_authenticator/lib/src/utils/validators.dart

```diff
@@ -11,7 +11,13 @@ import 'package:flutter/material.dart';
 */
 final usernameRegex = RegExp(r'^\S+$');
 final emailRegex = RegExp(r'^\S+@\S+$');
-final phoneNumberRegex = RegExp(r'^\+\d+$');
+
+/// Regex for a valid phone number.
+///
+/// The "+" prefix is excluded since validation is performed against the number
+/// without the prefix. That is, if the full number is "+1-123-555-7890", the
+/// "+1" is dropped prior to validation.
+final phoneNumberRegex = RegExp(r'^d+$');
```

Comment:
What is this meant to match? Should it not be `^[0-9]+$`?

Reply:
or `^\d+$`?

Reply:
Yes, the `\` should not have been removed. Updated.